akp/data_source_akp_kargo_schema.go

```diff
@@ -44,6 +44,11 @@ func getAKPKargoDataSourceAttributes() map[string]schema.Attribute {
 			MarkdownDescription: "Workspace name for the Kargo instance",
 			Computed:            true,
 		},
+		"kargo_resources": schema.ListAttribute{
+			MarkdownDescription: "List of Kargo resources",
```

Comment:
Could we add more details to this description and mention what resources are supported here?